src/engraving/dom/anchors.cpp

```diff
@@ -111,10 +111,14 @@ TimeTickAnchor* EditTimeTickAnchors::createTimeTickAnchor(Measure* measure, Frac
             continue;
         }
 
-        Staff* linkedStaff = toStaff(linkedObj);
-        Measure* linkedMeasure = linkedStaff->score()->tick2measureMM(measure->tick());
+        Staff* linkedStaff = linkedObj == staff ? staff : toStaff(linkedObj);
+        Measure* linkedMeasure = linkedObj == staff ? measure : linkedStaff->score()->tick2measureMM(measure->tick());
+        if (!linkedMeasure) {
+            return nullptr;
```

Comment:
This should `continue`, not `return nullptr`. Cause we may fail to create the anchor in the part, but we probably have created it successfully in the score, so we must let the function complete and return that.